.codecov.yaml

```diff
@@ -10,11 +10,6 @@ coverage:
       # only failing because of overall project coverage threshold.
       # See https://docs.codecov.io/docs/commit-status#disabling-a-status.
       default: false
-comment:
-  # Delete old comment and post new one for new coverage information.
-  # This is so that the code coverage info is closer to the newly push
-  # commits on GitHub PR UI.
-  behavior: new
```

Comment:
Will we still have the comment up to date on every new push (even when it is not the latest)?

Reply:
Yes, comment is up-to-date every new push. It gets "Edited" by the bot, so we will still see new information in UI.